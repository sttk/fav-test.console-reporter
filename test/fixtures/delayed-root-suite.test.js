'use stirct';

setTimeout(function() {
  describe('my suite', function() {
    it('my test', function() {
    });
  });

  run();
}, 5000);
