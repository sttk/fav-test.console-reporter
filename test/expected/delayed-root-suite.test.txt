

  my suite
    âœ“ my test


  1 passing (6ms)

